plugins/org.jkiss.dbeaver.model.sm/src/org/jkiss/dbeaver/model/security/SMObjects.java

```diff
@@ -19,6 +19,6 @@
 public interface SMObjects {
     String PROJECT_OBJECT_TYPE_ID = "project";
 
-    SMObjectType DATASOURCE = new SMObjectType("datasource");
-    SMObjectType PROJECT = new SMObjectType(PROJECT_OBJECT_TYPE_ID);
+    SMObjectType DATASOURCE = SMObjectType.datasource;
```

Comment:
Looks crazy. If we turned SMObjectType into enum then we don't need these constants anymore.

Reply:
Fixed. Constants were removed.